### PR TITLE
#72: fix styles for select in admin forms

### PR DESCRIPTION
**Pull request by kirillarutyunov**
image

It is ok now in Mac OS. Looks like need to test it on Win/Linux OS.

### CODE REVIEW

File: database/migrations/2021_09_13_183508_create_blog_subcategories_table.php [hunk 1]
```diff
@@ -14,7 +14,7 @@ class CreateBlogSubcategoriesTable extends Migration
     public function up()
     {
         Schema::create('blog_subcategories', function (Blueprint $table) {
-            $table->primary('id');
+            $table->z('id');
```
Comment (meinhoonharsh) [inline database/migrations/2021_09_13_183508_create_blog_subcategories_table.php:17]: Could you please tell me, what does z works here

Comment (kirillarutyunov) [reply database/migrations/2021_09_13_183508_create_blog_subcategories_table.php:17]: Sorry, I will fix it soon :)

**Comment by meinhoonharsh** [comment]
By the way, you've created a very fantastic PR

**Comment by kirillarutyunov** [comment]
@meinhoonharsh I've fixed the changes in migrations. It related to #76 issue.

**Pull request merged**